pytest.ini

```diff
@@ -1,7 +1,5 @@
 [pytest]
 log_cli = 1
 log_cli_level = INFO
-env =
```

Comment:
Why was this removed?

Reply:
pytest 6.1.1 does not support `env` (and report error), and I don't find the usage of this `env` either.

Reply:
This is to set the env var on the system to make boto3 happy and not fail. What is the new way to do this then?

Reply:
I will double check the new way. Do you know how to produce boto3 failure?

Reply:
I don't think the existing env variable `AWS_DEFAULT_REGION` is being set.
I am at develop branch cd4c061de406626480625b9a8da97ee6194914e1 and try to run a test after `make init`:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/3804518/96627960-745cde80-12c6-11eb-91ef-574775c8ec06.png">

Reply:
Interesting! Thanks for verifying this. Seems reasonable to remove then.

Reply:
https://github.com/MobileDynasty/pytest-env

I searched online and it seems never an option in pytest.ini. It requires a pytest plugin `pytest-env` which we don't have 😅